src/lib/components/payments/Checkout.svelte

```diff
@@ -58,8 +58,10 @@
               });
             },
             onError(err) {
-              collectError("payment error", err);
-              dispatch("paymentError");
+              if (err.message !== "Detected popup close") {
```

Comment:
If someone closes the pop up early it would throw an error from Paypal, but this should not result in showing the "Sorry, there was an error" screen because the user may just be switching to another method